### ISSUE TITLE
Don't assume footprints are in PropertyMap.shp

### ISSUE THREAD

**Issue by anderslogg** [dtcc-platform/dtcc-builder#41]
Proposal: Read all shp files present in the directory, same as for point cloud files

**Comment by dwastberg** [comment]
I agree that it is a problem that needs fixing. The problem with the approach you propose is that it will break as soon as we add road networks or land usage shp files.

**Comment by vbassn** [comment]
@dwastberg @anderslogg How about we make it a parameter? E.g. buildingsFileName we already have but how about  roadNetworkFilename etc?
